lib/tree.rb

```diff
@@ -29,19 +39,17 @@ def initialize
   end
 end
 
-class Apple <
-  attr_reader #what should go here 
+class Apple < Fruit
+  attr_reader :color, :diameter
 
-  def initialize(color, diameter)
+  def initialize()
+    @color = ['red'].sample
+    @diameter = rand(5..10)
```

Comment:
Why not just `@color = 'red'`?

Reply:
I meant to have @color = ['red','yellow','green'].sample, but I edited the array when I was testing it.